templates/base/cluster-template.yaml

```diff
@@ -116,9 +116,15 @@ spec:
           - localhost
           - 127.0.0.1
           - 0.0.0.0
+        extraArgs:
```

Comment:
is this required for kube-vip 5.5 upgrade?

Reply:
no this is for `Implement tls-cipher-suites enforcement`